app/views/templates/search/search_object.rb

```diff
@@ -175,100 +189,99 @@ def content
-              institution_name:    render_document_show_field_value(document, 'aggregations.edmDataProvider'),
+              url: render_document_show_field_value(document, 'aggregations.edmIsShownAt'),
+              institution_name: render_document_show_field_value(document, 'aggregations.edmDataProvider'),
               institution_country: render_document_show_field_value(document, 'europeanaAggregation.edmCountry'),
-              content_present:     collect_values(['aggregations.edmDataProvider', 'europeanaAggregation.edmCountry']).length > 0
             },
             people: data_section({
-              :title => 'site.object.meta-label.people',
-              :sections  => [
+              title: 'site.object.meta-label.people',
+              sections: [
                 {
-                  :title  => 'site.object.meta-label.creator',
-                  :fields => ['agents.prefLabel'],
-                  :collected => document.proxies.collect do |proxy|
-                                        res = []
-                                        val = proxy.fetch('dcCreator', nil)
-                                        val.each{|subject|
-                                          res << subject
-                                        } unless val.blank?
-                                        res
-                                    end,
-                  :url    => 'q',
-                  :extra  => [
+                  title: 'site.object.meta-label.creator',
+                  fields: ['agents.prefLabel'],
+                  collected: document.proxies.collect do |proxy|
+                      res = []
+                      val = proxy.fetch('dcCreator', nil)
+                      val.each{|subject|
+                        res << subject
+                      } unless val.blank?
+                      res
+                  end,
+                  url: 'q',
+                  extra: [
                       {
-                        :field  => 'agents.begin',
-                        :map_to =>  'life.from.short'
+                        field: 'agents.begin',
+                        map_to: 'life.from.short'
                       },
                       {
-                        :field  => 'agents.end',
-                        :map_to => 'life.to.short'
+                        field: 'agents.end',
+                        map_to: 'life.to.short'
                       }
                     ]
                 },
                 {
-                  :title   => 'site.object.meta-label.contributor',
-                  :fields  => ['proxies.dcContributor']
+                  title: 'site.object.meta-label.contributor',
+                  fields: ['proxies.dcContributor']
                 }
               ]
             }),
             places: data_section( {
-              :title => 'site.object.meta-label.place',
-              :sections  => [
+              title: 'site.object.meta-label.place',
```

Comment:
Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.